admin/admin.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package admin
+
+import (
+	"fmt"
+	"log/slog"
+	"net/http"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"github.com/vechain/thor/v2/log"
+)
+
+func HTTPHandler(logLevel *slog.LevelVar) http.Handler {
+	router := mux.NewRouter()
+	router.PathPrefix("/admin").Handler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
@MakisChristou IMO this should be a post request and the level should be declared in the request body.

GET requests shouldn't be used to update state


Maybe:
- GET `/admin/loglevel` to get the current log level
- POST `/admin/loglevel` with a body of `{ "level": "info" `

This allow us to create many endpoints on the admin domain if we have a use case to do so in the future

Reply:
Added this change